oci-unit-tests/helper/test_helper.sh

```diff
@@ -97,3 +100,61 @@ check_manifest_exists()
     debug "not found"
     return 1
 }
+
+# Check that the images listed
+# $1: List of all supported architectures to check.
+#     If empty, defaults to $SUPPORTED_ARCHITECTURES.
+check_supported_architectures()
+{
+    local supported_architectures="$1"
+    local registry
+    local utilsdir
+    local listed_architectures
+    local ret
+    ret=0
+
+    # We have to massage the registry name a bit.
+    case "${DOCKER_REGISTRY}" in
+	*"docker"*)
+	    debug "This function does not currently support the docker.io registry."
+	    return 0
+	    ;;
+	*"aws"*)
+	    registry="aws"
+	    ;;
+    esac
+
+    utilsdir="$(mktemp -d)/utils"
+    trap 'rm -rf ${utilsdir}' 0 INT QUIT ABRT PIPE TERM
+
+    debug -n "Checking if all supported architectures are present in the Docker manifest file"
```

Comment:
Is this by any chance a manifest list or an OCI image index? The reason I am asking is due to the "Accept" headers being passed to curl in the `https://git.launchpad.net/~canonical-server/ubuntu-docker-images/+git/utils` scripts, and to the fields being parsed with `jq` later in this commit.

If so, should the naming be replaced to reflect we are using manifest lists (or image indexes) instead of image manifests for (possibly) fetching the actual manifest for each architecture? Of course, if this is the case, this would probably go into another PR since other places may also refer to the manifest lists as manifests.

See https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list and https://docs.docker.com/registry/spec/manifest-v2-2/#image-manifest for further reference.

Reply:
Ah, I wasn't aware of the difference.  Based on the reference you gave, we're working with a manifest list.  I will update the naming to reflect that.  Thank you.